lib360dataquality/cove/threesixtygiving.py

```diff
@@ -139,6 +140,11 @@ def get_grants_aggregates(json_data):
 
             award_date = str(grant.get("awardDate", ""))
             if award_date:
+                try:
+                    year = award_date[:4]
+                    award_years[year] = award_years[year] + 1
```

Comment:
Why do you add one year?

Reply:
Ah yes, had to do a double-take on this myself, it's keeping a tally of grants for the `year` so the resulting `dict` will look like 

```python
award_years = {
"2002": 10,
"2005": 1,
"2006": 42,
}
```
etc

I've pushed a new commit that includes a comment to make this more obvious.

Reply:
ah!!! thanks for adding the comment.